rpc/rpc-wrappers.go

```diff
@@ -372,6 +374,23 @@ func NewRegistrationAuthorityServer(rpc Server, impl core.RegistrationAuthority,
 		return
 	})
 
+	rpc.Handle(MethodDeactivateAuthorization, func(ctx context.Context, req []byte) (response []byte, err error) {
+		var authz core.Authorization
+		err = json.Unmarshal(req, &authz)
+		if err != nil {
+			// AUDIT[ Error Conditions ] 9cc4d537-8534-4970-8665-4b382abe82f3
```

Comment:
I think we've been trying to delete these `// AUDIT` comments going forward since the tooling to look for them isn't in use anymore (Or am I remembering this wrong?)